http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20191125_stop_and_update.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
so here, in addition to a flag, I add a full updated MachineType to the db, as more or less a temporary variable. The need to do this was the motivation for this https://broadworkbench.atlassian.net/secure/RapidBoard.jspa?rapidView=35&projectKey=IA&view=planning&selectedIssue=IA-1530.  It may be a bit excessive to include all fields in MachineType, because only the updatedMasterMachineType is leveraged at the moment, but I wanted to ensure we could support other fields being updated for the Stop/Delete case if that became a priority suddenly.